Containerfile

```diff
@@ -57,7 +57,8 @@ RUN /prepare.sh "${prepare_args}"
 
 COPY --from=cache /build /usr/src/shuttle/
 
-# Any prepare steps that depend on the COPY from src cache
+# Any prepare steps that depend on the COPY from src cache.
+# This step currently installs shuttle-next and adds the panamax mirror config.
```

Comment:
nit: This is done for every Container that follows this structure, so the comment might be too specific? But tbh deployer is kind of the only user of prepare scripts.

Reply:
Oh, good point! I'll refactor it to say what this step does in the deployer.